packages/field-base/test/input-control-mixin.test.js

```diff
@@ -369,6 +370,18 @@ const runTests = (defineHelper, baseMixin) => {
         element._preventInputDebouncer.flush();
         expect(element.hasAttribute('input-prevented')).to.be.false;
       });
+
+      it('should check the helper checkbox', async () => {
+        element.allowedCharPattern = '';
+        const helperCheckbox = fixtureSync(`<input id="helper-checkbox" type="checkbox" slot="helper"/>`);
+        element.appendChild(helperCheckbox);
+        helperCheckbox.focus();
+
+        await sendKeys({ down: 'Space' });
+        await sendKeys({ up: 'Space' });
```

Comment:
```suggestion
        await sendKeys({ press: 'Space' });
```
Can just use `press` here.